src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap

```diff
@@ -253,19 +152,19 @@ exports[`PodcastExternalLinks Should render external links 1`] = `
           >
             Apple
```

Comment:
Would it be a good idea to add the lang attrributes as part of this PR too? https://github.com/bbc/simorgh/issues/8981

Reply:
Absolutely. Let me fix and merge https://github.com/bbc/simorgh/issues/8981 in here as well. Nice catch @greenc05

Reply:
@greenc05 this issue ha been addressed

Reply:
Yes, the lang attribute will update to English on english pages :) Thanks for the review @greenc05.